src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int ONE = 1;
+    private static final int ZERO = 0;
+
+    public String getStatistic(String fromFileName, String toFileName) {
+        File inputFile = new File(fromFileName);
+        File outputFile = new File(toFileName);
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(inputFile))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] split = value.split(",");
```

Comment:
comma, supply, buy and result should be constants

Reply:
https://github.com/mate-academy/jv-work-with-file/pull/1515/files#r1763685707 
comment not fixed